### PR TITLE
implemented internationalization

### PR DESCRIPTION
**Pull request by DeAlexandra**
-implemented internationalization with i18next library
-added translations folder for en and ro languages
-added language switcher in the header of the page

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -8,8 +8,6 @@ const Users = lazy(() => import("./pages/users/Users"));
 const ErrorPage = lazy(() => import("./pages/ErrorPage"));
 const Dashboard = lazy(() => import("./pages/dashboard/Dashboard"));
 
-
-
 export default function App() {
   return (
     <Router>
```
Comment (enfrbhd) [inline src/App.js:13]: please use translations on loading page message

File: src/components/LanguageSelect.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                <Select
+                    labelId="demo-simple-select-autowidth-label"
+                    id="demo-simple-select-autowidth"
+                    onChange={ handleChangeLang }
+                    autoWidth
+                    label="Language"
+                    defaultValue={ "en" }
+                    sx={ { color: "white" } }
+                >
+                    <MenuItem value="en" selected>English</MenuItem>
```
Comment (enfrbhd) [inline src/components/LanguageSelect.jsx:29]: please use translations also here: English and Romanian for English language (use ternary operator to check selected language) and Engleza, Romana for Romanian language

File: src/App.js (hunk 1 above)
Comment (DeAlexandra) [reply src/App.js:15]: added translation for loading page message

File: src/components/LanguageSelect.jsx (hunk 2 above)
Comment (DeAlexandra) [reply src/components/LanguageSelect.jsx]: added translations for language options according to the selected language

**Review by enfrbhd — APPROVED** [review]

**Pull request merged**